projects/cdk/services/theme-color.service.ts

```diff
@@ -0,0 +1,26 @@
+import {DOCUMENT} from '@angular/common';
+import {inject, Injectable} from '@angular/core';
+import {Meta} from '@angular/platform-browser';
+import {tuiCreateToken} from '@taiga-ui/cdk/utils';
+
+export const TUI_THEME_COLOR = tuiCreateToken<string | null>(null);
```

Comment:
Technically we can have it out of sync if somebody adds this meta tag to index.html and not provide the token. Maybe we should use `Meta` to try and get default value in case it is defined after all?

Reply:
fixed